modules/doc/content/help/inl/hpc_binary.md

```diff
@@ -0,0 +1,251 @@
+```
+
+### (2) Run the simulation
+
+In the previous section a test input file was created in your INL-HPC home directory within
+the "testing" folder. MOOSE-based applications are designed to be executed from the location
+of the input file, so start by navigating to this location.
+
+```bash
+cd ~/testing
+```
+
+Next, execute the application with the "-i" argument followed by the input filename. For
+example, when using BISON with the created test input file.
+
+```bash
+/apps/herd/bison/bison-opt -i input.i
+```
+
+### (3) Viewing Results id=viewing-results
+
+There are many ways to view results including options using the remote machine, please refer
+to the visualization systems linked on [hpcweb.hpc.inl.gov/hardware](https://hpcweb.hpc.inl.gov/hardware).
+
+However, in many cases it is desirable to view the results on your local machine. This is done
+by copying the content from the remote machine to your local machine using `scp`. For example,
+to copy the output of this test run use the following command on your local machine.
+
+```bash  style=background-color:#151B54;
+scp <your hpc user id>@hpclogin:~/testing/input_out.e /path/to/the/local/destination
+```
+
+## Scheduling Jobs with PBS
+
+This section will provide you with a basic example of using [!ac](PBS) for scheduling jobs to run in
+INL resources. For detailed information regarding using [!ac](PBS) please visit the [INL-HPC PBS
+website](https://hpcweb.hpc.inl.gov/home/pbs).
+
+In general, scheduling jobs requires two steps:
+
+1. create a [!ac](PBS) script that describes the required resources and the commands to execute and
+1. submit the script to the scheduler.
+
+### (1) Create [!ac](PBS) Script
+
+Using the example problem from above a simple script for scheduling may be created. The top
+portion of the script provides [!ac](PBS) directives that are passed to scheduler. Notice that these
+lines are simply the arguments that are being passed to the scheduler, as was done via the
+command line in the interactive job above.
+
+```bash
+#!/bin/bash
+#PBS -N test_run
+#PBS -l select=1:ncpus=48:mpiprocs=48
+#PBS -l walltime=5:00
+#PBS -P moose
+
+cd $PBS_O_WORKDIR
+source /etc/profile.d/modules.sh
+module load use.moose moose-apps PETSc BISON
+
+mpirun /apps/herd/bison/bison-opt -i input.i Mesh/uniform_refine=7
+```
+
+It is recommend that this script be created in the same location with the input file(s) it will be
+executed. This will ensure that the associated output from the simulation ends up in the same
+location. For this example the script is located in the testing folder: `~/testing/test.sh`.
+
+The [!ac](PBS) directives for this script include:
+
+1. `-N test_run` to set the job name;
+1. `-l select=1:ncpus=48:mpiprocs=48` indicates to utilize one compute (`select=1`), utilizing all
+   48 cores and run with 48 mpi processes;
+1. `-l walltime=5:00` allocates 5 minutes of time for the simulation; and
+1. `-P moose` dictates that the job should part of the MOOSE queue.
```

Comment:
I don't really have much time to go through this yet. I'll find that soon. One comment is that we should probably provide a list of projects or point them to a resource that has it. I think Eric/Tami like when people "charge" their time to the correct projects. There are projects for NEAMS, CASL, etc and we want to make sure hours go to the right places.

Reply:
There is a list on HPC pages that I can link to, I will add it after you go through the rest. I am sure you will have more to say.